Library/Formula/translate-shell.rb

```diff
@@ -0,0 +1,31 @@
+class TranslateShell < Formula
+  desc "Command-line translator using Google Translate and more"
+  homepage "https://www.soimort.org/translate-shell"
+  url "https://github.com/soimort/translate-shell/archive/v0.9.3.1.tar.gz"
+  sha256 "7b5a2404ead919570cfa4d741c521ed7e124bbf27d6f5de1a6a598e7e713c2a6"
+  head "https://github.com/soimort/translate-shell.git", :branch => "develop"
+
+  option "with-espeak", "Use eSpeak for Text-to-Speech"
+  option "with-mpg123", "Use mpg123 for Text-to-Speech"
+  option "with-mplayer", "Use mplayer for Text-to-Speech"
+  option "with-mpv", "Use mpv for Text-to-Speech"
+
+  depends_on "fribidi"
+  depends_on "gawk"
+  depends_on "rlwrap"
+
+  depends_on "espeak" => :optional
+  depends_on "mpg123" => :optional
```

Comment:
Nothing bad happens, one of them will be picked up. Looking at the source code https://github.com/soimort/translate-shell/blob/e215f74c564c34b887767cc0261a1832239fe341/include/Translate.awk#L30-L48 and https://github.com/soimort/translate-shell/blob/ccfc22241233cb68c5f2612e5aff17d74b581c30/include/Main.awk#L120-L133, the current order of preference is `mplayer` > `mpv` > `mpg123` (the previous ones are audio players) > (the following ones are speech synthesizers) `say` > `espeak`, so in fact even without any optional dependency, `trans` can still use `say` for Text-to-Speech, although you'll get gibberish with unsupported languages (for instance, every Chinese character is pronounced as "CJK Unified Ideograph" with `say`, at least on my system). Also, since `espeak` has lower precedence than `say`, it actually won't be picked up even when installed, but that's just an implementation detail.